src/notification/twilio.ts

```diff
@@ -0,0 +1,26 @@
+import {Link, Store} from '../store/model';
+import {Logger, Print} from '../logger';
+import {Config} from '../config';
+import twilio from 'twilio';
+
+const config = Config.notifications.twilio;
+
+const client = twilio(config.accountSid, config.authToken);
```

Comment:
@jef Sorry - only just seen this was merged. Didn't get GitHub email notifications for some reason.

I've seen reports of users having issues since this has been merged, saying that they are getting an error that Twilio credentials are needed even when they haven't set any of the Twilio options.

I think this line is the cause. We should move it so we are only initalizing the client inside `sendTwilioMessage`. I'll try and get a PR up later.

Reply:
@WillsB3 another user provided a fix in #349 which has been merged.

Reply:
Ah great, thanks.

Sorry for the trouble